src/main/java/org/jspecify/meta/Implies.java

```diff
@@ -87,17 +87,13 @@
 @Target(ANNOTATION_TYPE)
 public @interface Implies {
   /**
-   * The annotation types implied by this annotation type
-   * (usually one). May not include any annotation type that
-   * declares an attribute with no default value. Including this
-   * annotation type itself, or any annotation type transitively
-   * implied by another in the array, is redundant but harmless
-   * (a larger cycle of implication is acceptable). Any number
-   * of types can be given. Their order is irrelevant. A
-   * duplicate type in the array is redundant but harmless. If
-   * the array is empty (e.g., {@code @Implies({})
-   * @interface One}), this entire annotation is redundant
-   * but harmless.
+   * The annotation types implied by this annotation type (usually one). May not include any
```

Comment:
I got confused several times by "this annotation type". I see that you're referring to the annotation being annotated with `@Implies`, but I kept reading it as referring to `Implies` itself.

Reply:
Feel free to ignore if you don't have a better idea, and I'll try to remember to come back to it later.

Reply:
Ugh, yeah. I band-aided this with a parenthetical in two places but didn't repeat it for the silly `@return` clause (which I'm annoyed javadoc even cajoles me into adding).